output/cloud/expv2/output.go

```diff
@@ -29,8 +30,14 @@ type Output struct {
 	metricsFlusher  noopFlusher
 	periodicFlusher *output.PeriodicFlusher
 
-	collector          *collector
-	periodicCollector  *output.PeriodicFlusher
+	collector         *collector
+	periodicCollector *output.PeriodicFlusher
+
+	// TODO: rename; this is an old name from v1
+	// it now stops the entry point to buffering samples
+	//
+	// What about stopMetricCollection
+	// Does it sound better?
 	stopSendingMetrics chan struct{}
```

Comment:
It is renamed to `stopSamplesCollection` as suggested here: https://github.com/grafana/k6/pull/3085#discussion_r1201933374 because it respects what it does better.